src/features/courses/course-route.ts

```diff
@@ -251,7 +254,8 @@ router.get('/questions',
 
         const questions = await courseController.getQuestions({
             userId: userId,
-            courseTopicContentId: req.query.courseTopicContentId
+            courseTopicContentId: req.query.courseTopicContentId,
+            studentTopicAssessmentInfoId: req.query.studentTopicAssessmentInfoId ?? version?.id,
```

Comment:
This will cause an error to get thrown if `version` is null, right? So this should be fine?